src/main/java/ro/cristi/curs/validator/CarValidator.java

```diff
@@ -0,0 +1,64 @@
+package ro.cristi.curs.validator;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import lombok.experimental.UtilityClass;
+import ro.cristi.curs.exception.CarValidatorException;
+import ro.cristi.curs.model.Car;
+
+@UtilityClass
+public class CarValidator {
+
+    public static Boolean validateCar(Car car) throws CarValidatorException {
+        validateCarMakerOrModel(car.getMaker(), "maker");
```

Comment:
don't do that :) 2 different functions maybe with reuse of code from a common private function